core.go

```diff
@@ -14,10 +14,15 @@ func NewCore(cfg Configuration, factory SentryClientFactory) (zapcore.Core, erro
 		return zapcore.NewNopCore(), err
 	}
 
+	levelEnabler := cfg.Level
+	if cfg.EnableBreadcrumbs {
+		levelEnabler = cfg.BreadcrumbLevel
```

Comment:
why we replace it?

Reply:
The LevelEnabler determines whether messages should be passed to the Write function. Normally the breadcrumb level will be lower than the normal error event level. So we need to tell Zap to enable writes for the lower level.

Reply:
To be safe, I think it would be wise to check that the breadcrumbs level is lower than the "main" level and return an error if it's not. Otherwise, it will lead to some strange behavior.

Reply:
could we combine these 2 levels into 1 enabler?
something like this:
`
type LevelEnabler struct {
	zapcore.Level
	breadscrumbLevel zapcore.Level
}
func (l *LevelEnabler) Enabled(lvl zapcore.Level) bool {
	return l.Level.Enabled(lvl) || l.breadscrumbLevel >= lvl
}`